src/Xrd/XrdPollE.icc

```diff
@@ -79,9 +88,30 @@ XrdPoll *XrdPoll::newPoller(int pollid, int maxfd)
 XrdPollE::~XrdPollE()
 {
    if (PollTab) free(PollTab);
+   if (WaitFd >= 0) close(WaitFd);
    if (PollDfd >= 0) close(PollDfd);
 }
-  
+
+/******************************************************************************/
+/*                              A d d E v e n t F d                           */
```

Comment:
This should be AddWaitFD (looks like you must have changed your mind at some point).